core/src/main/java/hudson/model/Queue.java

```diff
@@ -866,7 +841,7 @@ private WaitingItem peek() {
      * @since 1.483
      */
     public List<Item> getApproximateItemsQuickly() {
```

Comment:
And https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly#L39